lib/lua/jah/ack.lua

```diff
@@ -2,41 +2,43 @@ local ControlSpec = require 'controlspec'
 local Formatters = require 'jah/formatters'
 local Ack = {}
 
-Ack.specs = {}
+local specs = {}
 
-Ack.specs.send = ControlSpec.DB:copy()
-Ack.specs.send.default = -60
+specs.send = ControlSpec.DB:copy()
+specs.send.default = -60
 
-Ack.specs.sample_start = ControlSpec.UNIPOLAR
-Ack.specs.sample_end = ControlSpec.new(0, 1, 'lin', 0, 1, "")
-Ack.specs.loop_point = ControlSpec.UNIPOLAR
+specs.sample_start = ControlSpec.UNIPOLAR
+specs.sample_end = ControlSpec.new(0, 1, 'lin', 0, 1, "")
```

Comment:
nit: (here and in the following), odd to see `'lin'` with single spaces and units (`""`) with double.